frontend/src/Dashboard/CrossingUpdatesPage/CrossingUpdates.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import CrossingMap from '../../Map/CrossingMap'; 
+import './CrossingUpdates.css';
+
+class CrossingUpdates extends Component {
+  state = {
+    activeTab: 'list'
+  }
+
+  changeTab(tab) {
+    this.setState({activeTab: tab});
+  }
+
+  render() {
+    return (
+      <div className="CrossingUpdates">
+        <div className="flexcontainer">
+          <div className="CrossingUpdates__tab--header">SHOW: </div>
+          <div className={this.state.activeTab === 'list' ? 'CrossingUpdates__tab--active' : 'CrossingUpdates__tab--inactive'}
+               onClick={this.changeTab.bind(this,'list')}
+          >LIST</div>
+          <div className="CrossingUpdates__tab--header"> | </div>
+          <div className={this.state.activeTab === 'map' ? 'CrossingUpdates__tab--active' : 'CrossingUpdates__tab--inactive'}
```

Comment:
since you're doing it twice, I could see this ternary being pulled out into a function...

```js
getTabClass(tab) {
  const baseClassName = "CrossingUpdates__tab--";
  const classModifier = this.state.activeTab === tab ? 'active' : 'inactive';

  return baseClassName + classModifier;
}

# usage:
<div className={getTabClass('map')} ... />
```